src/CoreWCF.BuildTools/src/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+
+namespace CoreWCF.BuildTools
+{
+    internal static class DiagnosticDescriptors
+    {
+        private static readonly DiagnosticDescriptor parentClassShouldBePartialError = new DiagnosticDescriptor(id: "COREWCF_0100",
+                                                                              title: "Parent class should be partial",
```

Comment:
Naming convention of private static fields is for them to start with s_

Reply:
Fixed with this commit https://github.com/CoreWCF/CoreWCF/pull/493/commits/62f7b20cd46341b794c9b7f3fdf92668a379a1c8